tests/test_core.py

```diff
@@ -1,13 +1,84 @@
 """Tests for turbopy/core.py"""
+import pytest
+import numpy as np
 from turbopy.core import Grid
 
 def test_grid():
     """Test initialization of the Grid class"""
-    N_grid = 8
-    grid_conf = {"N": N_grid,
+    n_grid = 8
+    grid_conf = {"N": n_grid,
                  "r_min": 0,
                  "r_max": 0.1}
     grid = Grid(grid_conf)
```

Comment:
Please create a pytest fixture with this Grid configuration.

Reply:
You can then use the fixture in all your tests, instead of repeating the code so many times.